packages/config/config/devices/0x001a/rf9640.json

```diff
@@ -16,47 +17,42 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Delayed OFF time",
-			"description": "Time in seconds to delay OFF",
-			"unit": "secs",
+			"label": "Delay Off Time",
 			"valueSize": 1,
+			"unit": "seconds",
 			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
+			"maxValue": 127,
+			"defaultValue": 10,
```

Comment:
did you double check those?

Reply:
Yes, we were wrong before. https://www.eaton.com/content/dam/eaton/products/wiring-devices-and-connectivity/wiring-devices/z-wave-plus/z-wave-plus-universal-smart-dimmer-accessory-dimmer-rf9640n-rf9642z-instruction-sheet.pdf